src/diffusers/models/modeling_utils.py

```diff
@@ -272,15 +272,6 @@ def save_pretrained(
 
         weights_name = SAFETENSORS_WEIGHTS_NAME if safe_serialization else WEIGHTS_NAME
 
-        # Clean the folder from a previous save
```

Comment:
This part of the code is unnecessary since we don't have sharded checkpoints yet